plugins/woocommerce/client/blocks/packages/components/text-input/stories/validated-text-input.stories.tsx

```diff
@@ -234,7 +236,7 @@ const Template: StoryFn< ValidatedTextInputProps > = ( args ) => {
 
 	useEffect( () => {
 		setInputValue( getFormattedValue( args.value ) );
-	}, [ args.value, args.customFormatter ] );
+	}, [ args.value, args.customFormatter, getFormattedValue ] );
```

Comment:
And I think `args.customFormatter` can be removed from this array of dependencies, no?